src/overlays/actors/ovl_En_Poh/z_en_poh.c

```diff
@@ -864,28 +863,28 @@ s32 EnPoh_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
     return false;
 }
 
-static s8 sLimbToBodyParts[OBJECT_PO_LIMB_MAX] = {
-    BODYPART_NONE,    // OBJECT_PO_LIMB_NONE
-    BODYPART_NONE,    // OBJECT_PO_LIMB_01
-    BODYPART_NONE,    // OBJECT_PO_LIMB_02
-    BODYPART_NONE,    // OBJECT_PO_LIMB_03
-    ENPOH_BODYPART_4, // OBJECT_PO_LIMB_04
-    ENPOH_BODYPART_5, // OBJECT_PO_LIMB_05
-    BODYPART_NONE,    // OBJECT_PO_LIMB_06
-    BODYPART_NONE,    // OBJECT_PO_LIMB_07
-    BODYPART_NONE,    // OBJECT_PO_LIMB_08
-    ENPOH_BODYPART_0, // OBJECT_PO_LIMB_09
-    ENPOH_BODYPART_1, // OBJECT_PO_LIMB_0A
-    BODYPART_NONE,    // OBJECT_PO_LIMB_0B
-    BODYPART_NONE,    // OBJECT_PO_LIMB_0C
-    BODYPART_NONE,    // OBJECT_PO_LIMB_0D
-    BODYPART_NONE,    // OBJECT_PO_LIMB_0E
-    BODYPART_NONE,    // OBJECT_PO_LIMB_0F
-    ENPOH_BODYPART_2, // OBJECT_PO_LIMB_10
-    BODYPART_NONE,    // OBJECT_PO_LIMB_11
-    BODYPART_NONE,    // OBJECT_PO_LIMB_12
-    ENPOH_BODYPART_3, // OBJECT_PO_LIMB_13
-    BODYPART_NONE,    // OBJECT_PO_LIMB_14
+static s8 sLimbToBodyParts[POE_LIMB_MAX] = {
+    BODYPART_NONE,    // POE_LIMB_NONE
+    BODYPART_NONE,    // POE_LIMB_ROOT
+    BODYPART_NONE,    // POE_LIMB_ROOT_WRAPPER
+    BODYPART_NONE,    // POE_LIMB_BOTTEM_CLOAK_ROOT
+    ENPOH_BODYPART_4, // POE_LIMB_BOTTEM_CLOAK
+    ENPOH_BODYPART_5, // POE_LIMB_TOP_CLOAK
+    BODYPART_NONE,    // POE_LIMB_LEFT_ARM_ROOT
+    BODYPART_NONE,    // POE_LIMB_LEFT_ARM_WRAPPER
+    BODYPART_NONE,    // POE_LIMB_LEFT_FOREARM_ROOT
+    ENPOH_BODYPART_0, // POE_LIMB_LEFT_FOREARM
+    ENPOH_BODYPART_1, // POE_LIMB_LEFT_UPPER_ARM
```

Comment:
```suggestion
    ENPOH_BODYPART_LEFT_FOREARM, // POE_LIMB_LEFT_FOREARM
    ENPOH_BODYPART_LEFT_UPPER_ARM, // POE_LIMB_LEFT_UPPER_ARM
```
Could you name the body parts that are in `sLimbToBodyParts`?